nautobot/ui/package.json

```diff
@@ -19,28 +19,34 @@
         "@rjsf/core": "^5.4.0",
         "@rjsf/utils": "^5.4.0",
         "@rjsf/validator-ajv8": "^5.4.0",
+        "@svgr/webpack": "^5.5.0",
         "@testing-library/jest-dom": "^5.16.5",
         "@testing-library/react": "^13.4.0",
         "@testing-library/user-event": "^13.5.0",
         "axios": "^1.2.1",
+        "camelcase": "^6.2.1",
         "framer-motion": "^9.0.3",
+        "identity-obj-proxy": "^3.0.0",
+        "prompts": "^2.4.2",
         "react": "^18.2.0",
+        "react-app-polyfill": "^3.0.0",
+        "react-dev-utils": "^12.0.1",
```

Comment:
Check whether these various dependencies are needed for prod or whether dev-only is preferable?